### ISSUE TITLE
[BUG] useShow does not query when given a specific resource name.

### ISSUE THREAD

**Issue by emehmet** [pankod/refine#2463]
Describe the bug
useShow does not query when given a specific resource name.

Expected behavior
useShow to query with the given resource name when the page is loaded or rendered

```
"@pankod/refine-antd": "^3.41.0",
"@pankod/refine-core": "^3.69.0",
```

```javascript
const { queryResult } = useShow({ resource: 'getReport' });
const { data, isLoading } = queryResult;
console.log('🚀 ~ file: show.tsx ~ line 52 ~ ShowReports ~ data', data);
```

**Comment by aliemir** [comment]
Hey @emehmet, sorry to hear that you are having a problem. Can you please provide more context on how to reproduce the issue or even better can you try to create a minimal reproduction in stackblitz or codesandbox?

**Comment by emehmet** [comment]
Hey @aliemir ,
[https://pankod-refine-kzsvvr--3000.local.webcontainer.io/posts/show/7](https://pankod-refine-kzsvvr--3000.local.webcontainer.io/posts/show/7)

Please click this link. The show page doesn't show anything.

**Comment by emehmet** [comment]
Hey @aliemir 
Thank you for your solution. I wish you good work :blush:

**Comment by aliemir** [comment]
Hey @emehmet, found the issue and sent a PR for it #2467. We'll do the release soon 🚀

**Comment by omeraplak** [comment]
Hey @emehmet ,
We've released `@pankod/refine-core@3.69.2`. Could you approve this fix?
